components/automate-cs-ocid/habitat/hooks/run

```diff
@@ -47,7 +61,11 @@ source "{{pkg.svc_config_path}}/scripts/set_env_vars.sh"
 # pls make sure export happens after the file is updated.
 export CHEF_SECRETS_DATA=$(cat config/private-chef-secrets.json)
 
-bundle exec bin/rake db:migrate
+secrets-helper exec \
```

Comment:
@RoyShravani Do we have any changes in chef-server/oc-id ? I see the `export DATABASE_URL=$output` in line 43...

Reply:
Do we still need the changes form line 64 to 67, as we are exporting the DATABASE_URL

Reply:
That's correct, we don't need them. Removed that bit.

Reply:
> @RoyShravani Do we have any changes in chef-server/oc-id ? I see the `export DATABASE_URL=$output` in line 43...

No, we don't need any changes in chef-server/oc-id